lib/canvas.js

```diff
@@ -44,6 +44,25 @@ Canvas.prototype.createJPEGStream = function (options) {
   return new JPEGStream(this, options)
 }
 
+/**
+ * The OffscreenCanvas.convertToBlob() method creates a Blob object representing
+ * the image contained in the canvas.
+ *
+ * @see https://developer.mozilla.org/en-US/docs/Web/API/OffscreenCanvas/convertToBlob
+ * @since NodeJS v18.0.0
+ */
+Canvas.prototype.convertToBlob = async function (options = {}) {
+  const type = options.type && FORMATS.includes(options.type)
+    ? options.type
+    : 'image/png'
+
+  return new Promise((resolve, reject) => {
+    this.toBuffer((err, buf) => {
+      err ? reject(err) : resolve(new Blob([buf], { type }))
+    }, type, options)
```

Comment:
Instead of passing all `options` thru, what do you think about just looking at the supported `quality`. So that people don't start depending on passing other options thru here? Since as far as I can tell from the specification, only `type` and `quality` is allowed? 🤔 

```js
  const quality = options.quality != null
    ? { quality: options.quality }
    : undefined
```

```suggestion
    }, type, quality)
```

Reply:
sure 👍